backend/app/api/endpoints/base/context.py

```diff
@@ -53,11 +53,16 @@ async def get_generative_contexts(model: GetGenerativeContextRequest):
 @router.post("/stream")
 async def stream_images(model_info: GetGenerativeContextRequest):
     async def event_generator():
-        for _ in range(2):
-            data = ContextService().get_generative_contexts(
+        iterations = 0
+        for _ in range(3):
```

Comment:
Maybe there's a chance to make this a configuration inside the artifacts of the yaml? That way if we want more or less images it's easier to change